mlflow/llama_index/signature.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from mlflow.models.signature import ModelSignature
+from mlflow.types.schema import ColSpec, Schema
+
+_logger = logging.getLogger(__name__)
+
+
+def _signature_has_required_params(signature: ModelSignature) -> bool:
+    """
+    PLACEHOLDER: validate that the signature supports the relevant inference type
+    """
+    return True
+
+
+def generate_signature_output(index, data, model_config=None, flavor_config=None, params=None):
+    """
+    PLACEHOLDER
+    """
+    # Lazy import to avoid circular dependencies. Ideally we should move _LlamaIndexModelWrapper
+    # out from __init__.py to avoid this.
+    from mlflow.llama_index import _LlamaIndexModelWrapper
+
+    return _LlamaIndexModelWrapper(
```

Comment:
However, a utility function within init that constructs this can avoid this import. Build and instance of _LlamaIndexModelWrapper via a private func within the init.py